src/components/Auth/Login.jsx

```diff
@@ -41,11 +43,20 @@ export class LoginComponent extends Component {
     return '';
   };
 
+  socialAuthLogin = (provider) => {
+    const { submitting, socialAuth } = this.props;
+
+    if (!submitting) {
+      socialAuth(provider);
+    }
```

Comment:
Why is this if still here?

Reply:
It slipped my mind, I just added ternary operator